app/controllers/users_controller.rb

```diff
@@ -1,6 +1,6 @@
 class UsersController < ApplicationController
-  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
-  before_action :correct_user, only: [:edit, :update]
+  before_action :logged_in_user, only: %i(index, edit, update, destroy)
+  before_action :correct_user, only: %i(edit, update)
```

Comment:
No need to add comma to separate the array element, just a space is ok here. 
```ruby
before_action :correct_user, only: %i(edit update)
``` 
Please change everywhere accordingly